apps/back-end/controllers/record-controller.js

```diff
@@ -0,0 +1,195 @@
+const { Prisma, PrismaClient } = require('@prisma/client')
+
+const prisma = new PrismaClient({ log: ['query'] })
+
+const recordController = {
+  // create a new record
+  postReocrd: async (req, res, next) => {
+    try {
+      // #swagger.tags = ['Expense Record']
+      const { title, amount, note, categoryId } = req.body
+      // uncommend after adding authentication process
+      // const userId = req.user.id
+
+      if (!title || !amount || !categoryId)
+        return res.json(
+          'missing title or amount or category id to create a new expense record'
+        )
+
+      await prisma.record.create({
+        data: {
+          title,
+          amount,
+          note,
+          categoryId,
+          userId: 13,
```

Comment:
We'd better work on the user system in the next PR so we can use the right `userId`.

Reply:
I'll add the authentication process in the next PR.